pkg/xds/envoy/clusters/v3/lb_configurer.go

```diff
@@ -31,6 +33,11 @@ func (e *LbConfigurer) Configure(c *envoy_cluster.Cluster) error {
 		c.LbPolicy = envoy_cluster.Cluster_LEAST_REQUEST
 
 		lbConfig := e.Lb.GetLeastRequest()
+		if lbConfig == nil || lbConfig.GetChoiceCount() == 0 {
```

Comment:
The way we handle things like this in new policies is that we use a pointer and detect if the value is really nil or actually set to 0. Not sure if this can be done like this in the old policies.

Reply:
Yes, I couldn't find a way to detect if someone sets 0 or if it returned the body. In this way, we shouldn't break how the API works.